packages/openapi-v3-types/test/unit/openapi-v3-spec-types.unit.ts

```diff
@@ -0,0 +1,97 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/openapi-v3-types
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '@loopback/testlab';
+import {
+  ExampleObject,
+  ReferenceObject,
+  DiscriminatorObject,
+  XMLObject,
+  ExternalDocumentationObject,
+  ISpecificationExtension,
+  createEmptyApiSpec,
+  OpenAPIObject,
+} from '../..';
+
+describe('openapi-v3-types unit tests', () => {
+  let emptySpec: OpenAPIObject;
+
+  beforeEach(createEmptySpec);
+
+  it('sets version 3 for an empty spec', () => {
+    expect(emptySpec.openapi).to.equal('3.0.0');
+  });
+
+  it('sets the spec info object', () => {
+    expect(emptySpec.info).to.be.an.Object();
+    expect(emptySpec.info.title).to.equal('LoopBack Application');
+    expect(emptySpec.info.version).to.equal('1.0.0');
+  });
+
+  it('creates an empty paths object', () => {
+    expect(emptySpec.paths).to.be.an.Object();
+    expect(emptySpec.paths).to.be.empty();
+  });
+
+  it('creates a default servers array', () => {
+    expect(emptySpec.servers).to.be.an.Array();
+    expect(emptySpec.servers).to.have.lengthOf(1);
+    const server = emptySpec.servers ? emptySpec.servers[0] : undefined;
+    expect(server).to.not.be.Undefined();
+    const serverUrl = server ? server.url : '';
+    expect(serverUrl).to.equal('/');
+  });
+
+  function createEmptySpec() {
+    emptySpec = createEmptyApiSpec();
+  }
+});
+
+/**
```

Comment:
I think it'd make things clearer that the classes below are tests if they were enclosed within a describe block. I'm not sure if a describe block is allowed to exist without any `it` blocks though

Reply:
I can see if that works